packages/core/admin/admin/src/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
shouldn't we have this at the monorepo level ?

Reply:
https://github.com/strapi/strapi/pull/12789#discussion_r891148004